java/configuration/src/main/java/sleeper/configuration/properties/instance/UserDefinedInstancePropertyImpl.java

```diff
@@ -154,6 +163,11 @@ public Builder propertyGroup(PropertyGroup propertyGroup) {
             return this;
         }
 
+        public Builder defafultProperty(SleeperProperty property) {
+            this.defaultProperty = property;
+            return this;
```

Comment:
I think this could default the validation predicate to the same as the one on the default property. Please could we unit test that?

Reply:
Addressed